libs/blocks/quiz-results/quiz-results.css

```diff
@@ -7,3 +7,59 @@
 .quiz-results.show {
   visibility: unset;
 }
+
+.section.phone-only,
+.section.tablet-portrait-up,.section.tablet-landscape-up, 
+.section.desktop-up,.section.big-desktop-up {
+  display: none;
+}
+
+@media screen and (max-width: 599px) { 
+  .section.section.phone-only {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 600px) {
+  .section.tablet-portrait-up {
+    display: block;
+  }
+
+  .section.tablet-landscape-up, 
+  .section.desktop-up,.section.big-desktop-up {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .section.tablet-landscape-up {
+    display: block;
+  }
+
+  .section.tablet-portrait-up,
+  .section.desktop-up,.section.big-desktop-up {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .section.desktop-up {
+    display: block;
+  }
+
+  .section.tablet-portrait-up,.section.tablet-landscape-up, 
+  .section.big-desktop-up {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1800px) {
```

Comment:
Where is 1800px coming from?

Reply:
was using the same ref we use in Dexter 
https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862#:~:text=to%20CSS%20or-,Less%20just%20the%20same.,-%40mixin%20for%2Dphone